packages/ckeditor5-html-support/docs/features/general-html-support.md

```diff
@@ -0,0 +1,212 @@
+---
+category: features
+modified_at: 2021-06-13
+updated_at: 2021-06-14
+---
+
+# General HTML support
+
+The general HTML support ("GHS") feature allows the user to easily enable HTML features that are not explicitly supported by any other dedicated CKEditor 5 plugins.
+
+Some examples of HTML features that can be easily enabled using general HTML support include:
```

Comment:
```suggestion
Some examples of HTML features that can be easily enabled using General HTML Support include:
```
Since we are using GHS abbreviation everywhere in the guide, I think it makes sense to use the upper case feature name.